src/screens/CreatePost/elements/SheetPrivacy.js

```diff
@@ -4,54 +4,54 @@ import {StyleSheet, Text, View} from 'react-native';
 import RBSheet from 'react-native-raw-bottom-sheet';
 
 import FlatListItem from '../../../components/FlatListItem';
-import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
+import {COLORS} from '../../../utils/theme';
 
 const SheetPrivacy = ({privacyRef, data, select, onSelect}) => (
-    <RBSheet
-      ref={privacyRef}
-      closeOnDragDown={true}
-      closeOnPressMask={true}
-      customStyles={{
-        container: styles.containerSheet,
-        draggableIcon: styles.draggableIcon,
-      }}>
-      <View style={styles.container}>
-        <Text style={styles.header}>Privacy</Text>
-        {data.map((value, index) => (
-          <FlatListItem
-            key={index}
-            value={value.label}
-            index={index}
-            select={select}
-            onSelect={onSelect}
-            icon={value.icon}
-            desc={value.desc}
-          />
-        ))}
-      </View>
-    </RBSheet>
-  );
+  <RBSheet
+    ref={privacyRef}
+    closeOnDragDown={true}
+    closeOnPressMask={true}
+    customStyles={{
+      container: styles.containerSheet,
+      draggableIcon: styles.draggableIcon
+    }}>
+    <View style={styles.container}>
+      <Text style={styles.header}>Privacy</Text>
+      {data.map((value, index) => (
+        <FlatListItem
+          key={index}
+          value={value.label}
+          index={index}
+          select={select}
+          onSelect={onSelect}
+          icon={value.icon}
+          desc={value.desc}
+        />
+      ))}
+    </View>
+  </RBSheet>
+);
 
 export default SheetPrivacy;
 
 const styles = StyleSheet.create({
   container: {
     paddingTop: 20,
-    paddingBottom: 38,
+    paddingBottom: 38
   },
   header: {
-    color: colors.black,
+    color: COLORS.black,
     fontFamily: fonts.inter[600],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property in the header style has been updated to use the `COLORS.black` constant from the `utils/theme` module instead of the previous `colors.black` constant. This change is consistent with the PR summary and seems to be a part of a larger refactoring effort to standardize color usage across the codebase.
```diff
- color: colors.black,
+ color: COLORS.black,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->